src/components/ResourcePage.tsx

```diff
@@ -20,6 +21,19 @@ import { GridContainer, MainArea } from './Layout/Layout';
 
 const NotFoundPage = loadable(() => import('../containers/NotFoundPage/NotFoundPage'));
 
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const PageContent = styled.div<{ maxWidth: number }>`
+  width: 100%;
+  margin-left: auto;
+  margin-right: auto;
+  padding-left: 24px;
+  padding-right: 24px;
```

Comment:
Gikk for denne stylingen for å få det tilsvarende sånn det var når vi brukte OneColumn